src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -329,18 +344,31 @@ endif::[]
 
 [#el_perm,reftext="EL-permission"]
 Elevate Level Permission (EL):: Any unsealed capability with its valid tag set to one that is loaded from memory has its <<el_perm>> cleared and its <<section_cap_level>> restricted to the authorizing capability's <<section_cap_level>> if the authorizing capability does not grant <<el_perm>>.
++
 If sealed, then only <<section_cap_level,CL>> is modified, <<el_perm>> is unchanged.
-
++
 This permission is similar to the existing <<lm_perm>>, but instead of applying to the <<w_perm>> on the loaded capability it restricts the <<section_cap_level,CL>> field.
 
 ifdef::cheri_v9_annotations[]
 NOTE: *CHERI v9 Note:* This permission does not exist in CHERI v9, but is similar to CHERIoT's _LoadGlobal_ permission, except that any _global_ capability implicitly grants _LoadGlobal_.
 endif::[]
 
+[#x_perm,reftext="X-permission"]
+Execute Permission (X, used to authorize instruction fetch):: Allow instruction execution.
```

Comment:
Moving this one feels like a regression to me. RWX (and perhaps C) are the most important permissions that people care about. You don't want to bury X beneath all the increasingly-obscure other permissions (LM, then all the levels stuff) IMO.

Reply:
ok, reverted